server/app/models.py

```diff
@@ -1178,15 +1178,17 @@ class FinalSubmission(Base):
     submission = ndb.KeyProperty(Submission)
     revision = ndb.KeyProperty(Submission)
     queue = ndb.KeyProperty(Queue)
-    submitter = ndb.KeyProperty(User) # TODO Change to ComputedProperty
+    server_time = ndb.ComputedProperty(lambda q: q.submission.get().server_time)
+    # submitter = ndb.ComputedProperty(lambda q: q.submission.get().submitter.get())
+    submitter = ndb.KeyProperty(User)
     published = ndb.BooleanProperty(default=False)
 
     @property
-    def server_time(self):
```

Comment:
I don't think we use FinalSubmission.server_time - but I'm not 100% sure. Is there a reason to remove it?